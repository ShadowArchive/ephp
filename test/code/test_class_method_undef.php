<?php

class A {}
$a = new A;
$a->no_exists();

