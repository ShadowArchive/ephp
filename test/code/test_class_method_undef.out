
Fatal error: Call to undefined method A::no_exists() in {{CWD}}/test/code/test_class_method_undef.php on line 5
